Snowflake.Data/Core/RestRequester.cs

```diff
@@ -103,7 +103,7 @@ public HttpResponseMessage Get(IRestRequest request)
 
             try
             {
-                var response = await HttpUtil.getHttpClient().SendAsync(request, linkedCts.Token).ConfigureAwait(false);
```

Comment:
When I reverted this line, the test case passed.

Reply:
Strange!
Are you sure that the failed test was not just some random network problem? Have you tried running the failed test again? Have you tried running the test locally?
As I said, it works on my machine...

Reply:
I have tried on my machine many times and it always pass.
However it consistently fail on appveyor.

Reply:
I have never used appveyor,  so I am afraid I cant help you with that.
But in any case using HttpCompletionOption.ResponseHeadersRead is the most important change in this pr. This option alone saves over 270 MB per prefetch slot. I think you should put some effort into understanding exactly what the problem is with running in appveyor.

Reply:
How much we could save just with changing from char array to byte array?

Reply:
Each reusable chunk stores all uncompressed data in the chunk. This is typically 140MB for the largest chunks I have seen.
If stored in a byte array as UTF8 this will use 140MB, if stored as char[] this will take two bytes per character or 280MB
So this change will save about 140MB per chunk